dist/js/foundation.d.ts

```diff
@@ -472,7 +472,7 @@ declare module FoundationSites {
 
     // utils
     Box: Box;
-    KeyBoard: KeyBoard;
+    Keyboard: Keyboard;
```

Comment:
We should add a breaking change warning for this. Because this might break existing code.

Reply:
I don't think. `KeyBoard` never worked and only `Keyboard` valid. It's not a change, it's a fix.

Reply:
If anyone uses this method and worked around this error through typecast it will create typescript compile errors.

Reply:
I updated the changelog.

> Fix Keyboard util TypeScript declaration from `KeyBoard` to `Keyboard`

If someone has a Typescript compilation error, they will easily find the reason.